src/guc.c

```diff
@@ -572,6 +573,17 @@ _guc_init(void)
 							 NULL,
 							 NULL);
 
+	DefineCustomBoolVariable(MAKE_EXTOPTION("enable_merge_on_cagg_refresh"),
+							 "Enable MERGE statement on cagg refresh",
+							 "Enable MERGE statement on cagg refresh",
+							 &ts_guc_enable_merge_on_cagg_refresh,
+							 false,
```

Comment:
why is the default false?

Reply:
Despite it is not a complicated change from technical point of view it touch a very critical code-path for cagg refresh and for the first release we want to have control of which customers will use it before turn it on by default for everyone.

Reply:
please change it to default true